static/js/rendered_markdown.js

```diff
@@ -166,6 +166,26 @@ export const update_elements = (content) => {
         }
     });
 
+    content.find("h4.poll-question-header").each(function () {
+        if ($(this).text().includes("<time:")) {
+            const text = $(this)
+                .text()
+                .match(/<time:(.*)>/)
+                .pop();
+            const time_string = $(this)
+                .text()
+                .match(/<time:(.*)>/)
+                .pop();
+            const rendered_time = timerender.render_markdown_timestamp(parseISO(time_string), text);
+            $(this).text(
+                $(this)
+                    .text()
+                    .replace(/<time:(.*)>/, ""),
+            );
+            $(this).append(`<time title="${rendered_time.title}">${rendered_time.text}</time>`);
```

Comment:
This HTML injection bug opens a cross-site scripting vulnerability.

Reply:
The alternative to this would be to add a markdown renderer to the handlebar template of the poll_widget? Is that right?

Reply:
I have fixed this in the below commits.